### PR TITLE
add message requesting a response from the cliente when someone try t…

### PR DESCRIPTION
**Pull request by matiascalegaris**
when a client tries to connected to a connecter char it send a ping request, on the next connection try if the client didn't respond after 10 secs we assume a connection issue and kick the connected user

### CODE REVIEW

File: Codigo/Protocol_Writes.bas [hunk 1]
```diff
@@ -3721,7 +3721,7 @@ Public Sub WriteDatosGrupo(ByVal UserIndex As Integer)
 
 124                     If i = 1 Then
 126                         Call Writer.WriteString8(UserList(UserList( _
-                                    .Grupo.Lider).Grupo.Miembros(i)).Name & "(Líder)")
+                                    .Grupo.Lider).Grupo.Miembros(i)).name & "(Líder)")
```
Comment (morgolock) [inline Codigo/Protocol_Writes.bas:3724]: revert to Name or rename field to name

File: Codigo/Protocol.bas [hunk 2]
```diff
@@ -19232,6 +19235,14 @@ On Error GoTo HandleDebugRequest_Err:
 HandleDebugRequest_Err:
 102     Call TraceError(Err.Number, Err.Description, "Protocol.HandleDebugRequest", Erl)
 End Sub
+
+Private Sub HandleReplyRequestResponse(ByVal UserIndex As Integer)
```
Comment (morgolock) [inline Codigo/Protocol.bas:19239]: Please rename to something like `HandleResponseConnectionPingRequest`

The current name is not very descriptive

File: Codigo/Protocol.bas [hunk 3]
```diff
@@ -19232,6 +19235,14 @@ On Error GoTo HandleDebugRequest_Err:
...
+On Error GoTo HandleReplyRequestResponse_Err:
+    UserList(UserIndex).ExpectPing = False
```
Comment (morgolock) [inline Codigo/Protocol.bas:19241]: Missing `exit sub` after `UserList(UserIndex).ExpectPing = False`

File: Codigo/Protocol_Writes.bas [hunk 4]
```diff
@@ -5511,6 +5511,21 @@ writeSensuiRetrasado_Err:
     Call TraceError(Err.Number, Err.Description, "Argentum20Server.Protocol_Writes.writeSensuiRetrasado", Erl)
     
 End Sub
+
+Public Sub WriteRequestResponse(ByVal UserIndex As Integer)
```
Comment (morgolock) [inline Codigo/Protocol_Writes.bas:5515]: Please rename to something more descriptive

File: Codigo/Protocol.bas [hunk 5]
```diff
@@ -533,7 +534,7 @@ Public Enum ClientPacketID
     PublicarPersonajeMAO
     EventoFaccionario    '/EVENTOFACCIONARIO
     RequestDebug '/RequestDebug consulta info debug al server, para gms
-    
+    ReplyRequestResponse
```
Comment (morgolock) [inline Codigo/Protocol.bas:537]: Rename to something that is more explanatory

**Review by morgolock — CHANGES_REQUESTED** [review]

File: Codigo/Protocol_Writes.bas (hunk 1 above)
Comment (matiascalegaris) [reply Codigo/Protocol_Writes.bas:3724]: the field its already called name, thats why vb6 keep changing it

**Pull request closed without merging**